acceptance/bundle/run/scripts/databricks-cli/target-is-passed/from_flag/script

```diff
@@ -0,0 +1,10 @@
+# Replace placeholder with an actual host URL and token
+envsubst < .databrickscfg > out && mv out .databrickscfg
+export DATABRICKS_CONFIG_FILE=.databrickscfg
+
+# Credentials will be picked up from .databrickscfg. Unset existing credentials.
+unset DATABRICKS_HOST
+unset DATABRICKS_TOKEN
+
+# Explicitly select oauth target
+trace $CLI bundle run validate -t oauth | jq .bundle.target
```

Comment:
Can you describe the intent for these tests?

I see the request log but don't see how it is relevant to the expectations.

Reply:
Added a comment explaining. We want to assert that the right target and auth credentials are used by the child command.